.pipelines/dwsql-pipelines.yml

```diff
@@ -215,4 +215,5 @@ jobs:
     inputs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)\Verify'
       ArtifactName: 'Verify'
-      publishLocation: 'Container'
\ No newline at end of file
+      publishLocation: 'Container'
+      
```

Comment:
```suggestion
      
 
```

nit: new line, not sure why but git UI complains last line in the file is missing